components/sponsors/SponsorModal.vue

```diff
@@ -4,15 +4,15 @@
             <div class="lightBox__closeButton" @click="close">✕</div>
             <div class="lightBox__title">
                 <sponsor-card :logo-url="context.logo_url" small></sponsor-card>
-                <h1 class="text-xl mx-8 border-b-8 border-solid">
+                <h1 class="text-xl mx-8">
                     {{ context.name }}
                 </h1>
             </div>
-            <h2 class="text-xl text-yellow-500 mt-4">{{ $t('about') }}</h2>
+            <h2 class="lightBox__subtitle text-xl mt-6">{{ $t('about') }}</h2>
             <div class="lightBox__intro">
                 <p class="text-sm">{{ getAttributeByLocale('intro') }}</p>
             </div>
-            <div class="lightBox__buttons">
+            <div class="lightBox__buttons sm:mt-20">
```

Comment:
I don't think we need `mt-20 (5rem)` => `100px`... I guess you were trying to increase the margin for the mobile view? If so, please use different margins for different breakpoints. Also, please test it on different resolutions locally before pushing the branch.
> This change actually may need to tweak a little bit of the layout, image, font size so that there's enough space for the description on mobile views. However, this may be out of the scope of this pr. You could decide to update it all at once or open another pr.

<img width="725" alt="Screen Shot 2021-07-04 at 9 15 31 PM" src="https://user-images.githubusercontent.com/7352613/124386777-91c7c080-dd0e-11eb-8327-5ba1419a9dd0.png">
<img width="593" alt="Screen Shot 2021-07-04 at 9 30 11 PM" src="https://user-images.githubusercontent.com/7352613/124386886-0a2e8180-dd0f-11eb-8316-fa4d07032cd4.png">